package.json

```diff
@@ -139,6 +139,8 @@
     "flow-check-ios": "flow check",
     "flow-check-android": "flow check --flowconfig-name .flowconfig.android",
     "lint": "eslint .",
+    "clang-format": "clang-format -i --glob=*/**/*.{h,cpp,m,mm}",
```

Comment:
Are we sure that in this configuration clang-formant will pick up the style guide from the file in the root of the repo (not some default settings)?

I suspect we should pass something like `--file`.

Reply:
I tested it, it uses the root config. Do you want me make it explicit ?

Reply:
> clang-format supports two ways to provide custom style options: directly specify style configuration in the -style= command line option or use -style=file and put style configuration in the .clang-format or _clang-format file in the project directory.

From https://clang.llvm.org/docs/ClangFormatStyleOptions.html.

But maybe adding it good for fb internal development ?